src/nft/components/details/AssetPriceDetails.tsx

```diff
@@ -96,6 +96,7 @@ const MarketplaceIcon = styled.img`
 `
 
 const BuyNowButton = styled.div<{ assetInBag: boolean; margin: boolean; useAccentColor: boolean }>`
+  position: relative;
```

Comment:
not getting the new hover state

Reply:
I am getting it on all browsers - it should be really subtle

Reply:
can see it in light mode but personally can't tell on dark, that's on design though